playground/src/plugins/signature.ts

```diff
@@ -57,9 +57,9 @@ export const signature: Plugin<Signature> = {
   pdf: image.pdf,
   propPanel: {
     schema: {},
-
     defaultSchema: {
       type: 'signature',
+      // icon: '<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-pen"><path d="M17 3a2.85 2.83 0 1 1 4 4L7.5 20.5 2 22l1.5-5.5Z"/></svg>',
```

Comment:
do you intend to leave this commented?

Reply:
Yes, this is example if not set icon, left sidebar icon will be plugin's Key text